configs/11.0/specs/monolithic_cross.spec

```diff
@@ -113,11 +113,11 @@ for INFO in $(ls ${RPM_INSTALL_PREFIX}/%{tgtinfodir_r}/*.info.gz); do
 	install-info ${INFO} ${RPM_INSTALL_PREFIX}/%{tgtinfodir_r}/dir
 done
 if [[ -w /usr/share/modules/modulefiles/. ]]; then
-	ln -sf %{_datadir}/modules/modulefiles/%{at_dir_name}-%{target} \
+  ln -sf ${RPM_INSTALL_PREFIX}/%{datadir_r}/modules/modulefiles/%{at_dir_name}-%{target} \
```

Comment:
The tab was replaced by two spaces and the line still passes the 80th column.
You could create a variable with "${RPM_INSTALL_PREFIX}/%{datadir_r}" and keep the tab and the column limit.